x-pack/plugins/security_solution/public/overview/components/detection_response/user_alerts_table/user_alerts_table.tsx

```diff
@@ -0,0 +1,174 @@
+
+import React, { useMemo } from 'react';
+
+import {
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiButton,
+  EuiEmptyPrompt,
+  EuiHealth,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { SecurityPageName } from '../../../../app/types';
+import { HeaderSection } from '../../../../common/components/header_section';
+import { useQueryToggle } from '../../../../common/containers/query_toggle';
+import { useNavigation, NavigateTo, GetAppUrl } from '../../../../common/lib/kibana';
+import * as i18n from '../translations';
+import { SEVERITY_COLOR } from '../util';
+import { AlertSeverityCounts, useUserAlertsItems } from './user_alerts_items';
+
+type GetTableColumns = (params: {
+  getAppUrl: GetAppUrl;
+  navigateTo: NavigateTo;
+}) => Array<EuiBasicTableColumn<AlertSeverityCounts>>;
+
+const DETECTION_RESPONSE_USER_SEVERITY_QUERY_ID = 'vulnerableUsersBySeverityQuery';
+
+export const UserAlertsTable = React.memo(() => {
+  const { getAppUrl, navigateTo } = useNavigation();
+  const { toggleStatus, setToggleStatus } = useQueryToggle(
+    DETECTION_RESPONSE_USER_SEVERITY_QUERY_ID
+  );
+  const { data, isLoading } = useUserAlertsItems({
+    skip: !toggleStatus,
+    queryId: DETECTION_RESPONSE_USER_SEVERITY_QUERY_ID,
+  });
+
+  const columns = useMemo(
+    () => getTableColumns({ getAppUrl, navigateTo }),
+    [getAppUrl, navigateTo]
+  );
+
+  return (
+    <EuiPanel hasBorder data-test-subj="userSeverityAlertsPanel">
+      <HeaderSection
+        id={DETECTION_RESPONSE_USER_SEVERITY_QUERY_ID}
+        title={i18n.USER_ALERTS_SECTION_TITLE}
+        titleSize="s"
+        hideSubtitle
+        toggleStatus={toggleStatus}
+        toggleQuery={setToggleStatus}
+      />
+
+      {toggleStatus &&
+        (isLoading || data.length > 0 ? (
+          <>
+            <EuiBasicTable
+              data-test-subj="userAlertsTable"
+              columns={columns}
+              items={data}
+              loading={isLoading}
+              noItemsMessage={<>{'No alerts found'}</> /** TODO */}
+            />
+            <EuiSpacer size="m" />
+            <EuiButton
+              onClick={() => navigateTo({ url: getAppUrl({ deepLinkId: SecurityPageName.users }) })}
+            >
+              {i18n.VIEW_ALL_USER_ALERTS}
+            </EuiButton>
+          </>
+        ) : (
+          <EuiEmptyPrompt title={<h3>{i18n.NO_ALERTS_FOUND}</h3>} titleSize="xs" />
+        ))}
+    </EuiPanel>
+  );
+});
+
+UserAlertsTable.displayName = 'UserAlertsTable';
+
+const getTableColumns: GetTableColumns = ({ getAppUrl, navigateTo }) => [
+  {
+    field: 'userName',
+    name: i18n.USER_ALERTS_USERNAME_COLUMN,
+    truncateText: true,
+    textOnly: true,
+
+    render: (userName: string) => {
+      const url = getAppUrl({ deepLinkId: SecurityPageName.users, path: userName });
+
+      return (
```

Comment:
Found that we have `UserDetailsLink` in x-pack/plugins/security_solution/public/common/components/links/index.tsx, could we reuse that here?

Reply:
awesome, thank you!